### PR TITLE
Fix FAQ switch category bug

### PR DESCRIPTION
**Pull request by alidevjimmy**
This PR fixes FAQ page switch between categories bug.

### CODE REVIEW

File: layouts/about/faq.html [hunk 1]
```diff
@@ -49,7 +49,7 @@ <h1 class="md:text-4xl text-2xl text-bold text-neutral-50 font-medium text-left"
     {{ $titleKey := $item.title }}
     <div class="p-px bg-gradient-to-b to-[#393939] from-[#C3C3C3] rounded-xl faq-btn-parent">
       <input class="btn btn-lg md:text-lg w-full h-full bg-[#141B25] !rounded-xl no-animation faq-btn" type="radio"
-        name="groups" aria-label="{{ $titleKey }}" {{ if eq $i 0 }}checked{{ end }} aria-value="{{ $item.title }}" />
+        name="groups" aria-label="{{ $titleKey }}" {{ if eq $i 0 }}checked{{ end }} aria-value="{{ lower $item.title }}" />
```
[the remark this answers is not in the document]
Comment (alidevjimmy) [reply layouts/about/faq.html:52]: 1- I used aria-value to track clicks on groups to show group content.
2- I don't understand what you mean about logic in `{{ }}`!
3- I changed the origin of my repository and closed previous PR so this PR is duplicated and will resolve quickly.

[the remark this answers is not in the document]
Comment (b00f) [reply layouts/about/faq.html:52]: 1. I understand, but it's better not to use `aria-value`. This seat is reserved for them. I'm sure you can find a better way.

2. Defining variables outside `{{ }}` gives them a wider scope, which helps with reusability.

3. A good title for the PR makes it easier to track changes. Please try to use a semantic PR title.

**Comment by alidevjimmy** [comment]
@b00f please check again.

**Review by b00f — APPROVED** [review]

**Comment by b00f** [comment]
Thanks @alidevjimmy

**Pull request merged**